playbooks/ansible/roles/sit.gpfs/tasks/server/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Disable firewall
+  service:
+    name: firewalld
+    enabled: false
+    state: stopped
```

Comment:
Is this a requirement from Storage Scale?

Reply:
No, but I would prefer to address firewall configuration in another patch. To configure it correctly, each node requires a different set of open ports, depending on what services are running and what role has each node inside the Storage Scale cluster. With the current approach, where some playbooks are only run on subsets of the servers, making the firewall configuration correctly could be a bit of a mess.